openstack/02a-MAClow/IEEE802154E.c

```diff
@@ -2045,8 +2067,10 @@ void notif_sendDone(OpenQueueEntry_t* packetSent, owerror_t error) {
 void notif_receive(OpenQueueEntry_t* packetReceived) {
    // record the current ASN
    memcpy(&packetReceived->l2_asn, &ieee154e_vars.asn, sizeof(asn_t));
-   // indicate reception to the schedule, to keep statistics

```

Comment:
That flag is set TRUE at the beginning of slot when an unscheduled EB will be sent, or when receiving the first EB (look ieee154e_processIE). It is used to avoid having statistics showed for cells that will usually be OFF.